### PR TITLE
Be 01 add cooking time

### PR DESCRIPTION
**Pull request by fatality1922**

### CODE REVIEW

File: app/models/recipe.rb [hunk 1]
```diff
@@ -1,3 +1,9 @@
 class Recipe < ApplicationRecord
   validates :name, presence: true
+  validates :cooking_time,  :numericality => { 
+    #   :only_integer => true,
```
Comment (PatrykTyszka) [inline app/models/recipe.rb:4]: remove comment

File: app/models/recipe.rb [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+      :greater_than => 1,
+      :less_than    => 1000,
```
Comment (PatrykTyszka) [inline app/models/recipe.rb:6]: Do we have this in Acceptance Criteria?

File: app/serializers/recipe_serializer.rb [hunk 3]
```diff
@@ -1,3 +1,9 @@
 class RecipeSerializer < ActiveModel::Serializer
-  attributes :id, :name, :content
+  attributes :id, :name, :content, :cooking_time
+
+    def cooking_time
+        return 'unknown' unless object.cooking_time
+        
+        object.cooking_time
```
Comment (PatrykTyszka) [inline app/serializers/recipe_serializer.rb:7]: 1. wrong indentation
2. please use const

File: spec/factories/recipes.rb [hunk 4]
```diff
@@ -2,5 +2,8 @@
   factory :recipe do
     name { Faker::Food.dish }
     content { Faker::Food.description }
+    cooking_time {Faker::Number.number(digits: 3) }
```
Comment (PatrykTyszka) [inline spec/factories/recipes.rb:5]:
```suggestion
    cooking_time { Faker::Number.number(digits: 3) }
```

File: spec/factories/recipes.rb [hunk 5]
```diff
@@ -2,5 +2,8 @@
...
   end
 end
+
+# test
```
Comment (PatrykTyszka) [inline spec/factories/recipes.rb:9]: ?

File: spec/controllers/api/v1/recipes_controller_spec.rb [hunk 6]
```diff
@@ -45,7 +46,7 @@
     context 'with invalid params' do
       it 'doesn\'t create recipe' do
         expect do
-          post :create, params: { recipe: { name: '', content: '' } }
+          post :create, params: { recipe: { name: '', content: '', cooking_time: '' } }
```
Comment (PatrykTyszka) [inline spec/controllers/api/v1/recipes_controller_spec.rb:49]: you did not change anything. This spec does not create recipe even without cooking_time

**Review by PatrykTyszka — CHANGES_REQUESTED** [review]

File: app/models/recipe.rb (hunk 2 above)
Comment (fatality1922) [reply app/models/recipe.rb:6]: it was agreed with front-end

File: app/serializers/recipe_serializer.rb [hunk 7]
```diff
@@ -1,9 +1,10 @@
 class RecipeSerializer < ActiveModel::Serializer
   attributes :id, :name, :content, :cooking_time
+  UNKNOWN = 'unknown'.freeze
```
Comment (PatrykTyszka) [inline app/serializers/recipe_serializer.rb:3]: consts before attributes

File: app/models/recipe.rb [hunk 8]
```diff
@@ -1,3 +1,8 @@
 class Recipe < ApplicationRecord
   validates :name, presence: true
+  validates :cooking_time,  :numericality => {
```
Comment (PatrykTyszka) [inline app/models/recipe.rb:3]: missing spec for this validation. Please add it

**Review by PatrykTyszka — CHANGES_REQUESTED** [review]

File: spec/serializers/recipe_serializer_spec.rb [hunk 9]
```diff
@@ -7,4 +7,20 @@
 
   it { is_expected.to include(name: recipe.name) }
   it { is_expected.to include(content: recipe.content) }
+  it { is_expected.to include(cooking_time: recipe.cooking_time) }
+  it { is_expected.to include(cooking_time: Integer) }
+  
+end
+
+describe RecipeSerializer do
```
Comment (PatrykTyszka) [inline spec/serializers/recipe_serializer_spec.rb:15]: ? why

Comment (PatrykTyszka) [reply spec/serializers/recipe_serializer_spec.rb:15]: just add to the existing class

File: spec/serializers/recipe_serializer_spec.rb [hunk 10]
```diff
@@ -7,4 +7,20 @@
...
+
+  let(:recipe) { build(:recipe, cooking_time: cooking_time) }
+  subject(:serialized_recipe) { described_class.new(recipe).to_h }
+
+  context 'when cooking_time value is nil' do
```
Comment (PatrykTyszka) [inline spec/serializers/recipe_serializer_spec.rb:20]: where is the rest, I saw more cases in yourr validation

**Review by PatrykTyszka — CHANGES_REQUESTED** [review]

File: spec/serializers/recipe_serializer_spec.rb [hunk 11]
```diff
@@ -1,26 +1,43 @@
 require 'rails_helper'
 
 describe RecipeSerializer do
-  subject(:serialized_recipe) { described_class.new(recipe).to_h }
```
Comment (PatrykTyszka) [inline spec/serializers/recipe_serializer_spec.rb:4]: this empty line is not necessary :D

**Review by PatrykTyszka — CHANGES_REQUESTED** [review]
You added specs for Serializer 👍  but still missing specs for the model.

**Comment by PatrykTyszka** [comment]
fix conflicts

**Review by PatrykTyszka — APPROVED** [review]

**Pull request merged**